File: java/Memory.java
public class Memory extends Hardware{
    
    private String frequency;
    private String memorySize;
    private String supportsCuda;
    
    public Memory(){
    }

    public Memory(String f, String ms, String sc){
        this.frequency = f;
        this.memorySize = ms;
        this.supportsCuda = sc;
    }

    public String getFrequency() {
        return frequency;
    }
    
    public String getSupportsCuda() {
        return supportsCuda;
    }
    
    public void setSupportsCuda(String supportsCuda) {
        this.supportsCuda = supportsCuda;
    }
    
    public String getMemorySize() {
        return memorySize;
    }
    
    public void setMemorySize(String memorySize) {
        this.memorySize = memorySize;
    }
    
    public void setFrequency(String frequency) {
        this.frequency = frequency;
    }

    public void showMemory(){
        System.out.println("Frequency     : " + frequency);
        System.out.println("Memory Size   : " + memorySize);
        System.out.println("Supports Cuda : " + supportsCuda);
    }

    
}

File: python/Memory.py
from Hardware import Hardware

class Memory(Hardware):

    #contructor
    def __init__(self):
        self.frequency = ""
        self.memorySize = ""
        self.SupportsCuda = ""
    
    def setFrequency(self, freq):
        self.frequency = freq

    def getFrequency(self):
        return self.frequency

    def setMemorySize(self, ms):
        self.memorySize = ms

    def getMemorySize(self):
        return self.memorySize

    def setSupportsCuda(self, sc):
        self.SupportsCuda = sc
    
    def getSupportsCuda(self):
        return self.SupportsCuda
    
    # Menampilkan isi Atribut
    def showMemory(self):
        print("Frequency     : " + str(self.frequency))
        print("Memory Size   : " + str(self.memorySize))
        print("Supports Cuda : " + str(self.SupportsCuda))